modules/oe_theme_content_document_reference/oe_theme_content_document_reference.info.yml

```diff
@@ -0,0 +1,14 @@
+name: OpenEuropa Theme Document reference
+type: module
+description: Companion module for the OE Content Entity Document reference module
+package: OpenEuropa
+core: 8.x
+dependencies:
+  - oe_content:oe_content_sub_entity_document_reference
+  - oe_theme:oe_theme_content_consultation
```

Comment:
`oe_theme:oe_theme_content_consultation` isn't a dependency. Please remove the line.
oe_theme:oe_theme_content_publication is a dependency because it ships teaser view mode for publication